scripts/irods/test/test_misc.py

```diff
@@ -477,3 +479,33 @@ def test_logical_path_including_the_AND_keyword_SPACES_and_a_SINGLE_QUOTATION_MA
         finally:
             if os.path.exists(filename):
                 os.unlink(filename)
+
+    @unittest.skipIf(test.settings.RUN_IN_TOPOLOGY, 'Only requires a single server')
+    def test_configuration_validation_reports_fqdn_size_violations__issue_8287(self):
+        config = IrodsConfig()
```

Comment:
consider leaving this link here in a comment to explain the 253...

https://web.archive.org/web/20190518124533/https://devblogs.microsoft.com/oldnewthing/?p=7873

Reply:
Done.